torch/_inductor/codecache.py

```diff
@@ -1637,7 +1641,9 @@ def _to_bytes(t: torch.Tensor) -> bytes:
                 return bytes(raw_array.contents)
 
             aot_constants = b"".join(
-                _to_bytes(tensor) for tensor in graph.constants.values()
+                _to_bytes(tensor)
+                for name, tensor in graph.constants.items()
+                if name not in graph.folded_constants
```

Comment:
Can you say more abou this ?

Reply:
When we are compiling the constants into the .so, we have 2 kinds of constants. 
Recall we have 2 steps, 1) split out the constant graph 2) rewire the main graph
In step 2), to successfully to have the compilation succeed, we actually have constants attached on the main graph (which, are the constants resulting from step 1), the output of `constant_graph.run()`.) The constants being attached are the "folded constants", which we do not want to include in the final .so, since they can be determined during "runtime" (which is the purpose of this change.)

Reply:
We only have one single .so being generated. So the .so file includes all the constants. The aot_constants here is constants that are being written into .so, so it doesn't matter whether the constant belongs to const graph or main graph, they are all in the single file, it's just the part of the code that want to use the constant need to grab it from the corresponding offset.